compiler/fir/analysis-tests/testData/resolve/varargInPrimaryConstructor.txt

```diff
@@ -5,7 +5,7 @@ FILE: varargInPrimaryConstructor.kt
         }
 
         public final val strings: R|kotlin/Array<out kotlin/String>| = R|<local>/strings|
-            public get(): R|kotlin/Array<out kotlin/String>|
+            public get(): R|kotlin/String|
```

Comment:
I think this change is not very much correct, getter of the property should also have an array type

Reply:
Fixed getter types